src/global/tracking/tracking.cc

```diff
@@ -3,10 +3,9 @@
 //
 //
 
-#include <Acts/Propagator/Navigator.hpp>
-
 #include <JANA/JApplication.h>
 #include <JANA/JEvent.h>
+#undef alignas
```

Comment:
Let's maintain forward-compatibility:
```suggestion
// Workaround until https://github.com/JeffersonLab/JANA2/pull/239
#ifdef alignas
#undef alignas
#endif
```

Reply:
"`#undef` has no effect if the name is not a macro." (https://gcc.gnu.org/onlinedocs/cpp/Undefining-and-Redefining-Macros.html)

Reply:
This commit was mainly intended as a test to see if this approach even resolves the issue with running here. I think having to add `#undef` everywhere does not scale.

Reply:
Test succeeded, by the way: https://github.com/eic/EICrecon/actions/runs/6102136482/job/16560729578?pr=922#step:5:862. 

So, at least adding `#undef` everywhere would be a solution. But then we need to add it **everywhere**.

Reply:
I think the better approach is to continue our separation of JANA2 and Acts. What is ultimately the reason for the issue in this file (tracking.cc) is that `TrackerSourceLinker_factory` needs to include the header for the type of data it produces. That is still an Acts structure, not podio. So, we end up having to include a bunch of Acts headers, which is what ultimately leads to the problem. If we turn `TrackerSourceLinks` into podio types (see EDM4hep and its plugin in the newer Acts versions) then we avoid this problem because then JANA2 and Acts headers don't ever need to share a translation unit.

Reply:
We can potentially forward declare our way out of this too, but I wasn't successful with that last night, before I really understood this issue.

Reply:
Implemented in #933. We can now be sure `alignas` is not defined.

Reply:
```suggestion
```